Assets/Scripts/UI/MouseOverFurnitureTypeText.cs

```diff
@@ -31,19 +31,26 @@ void Start()
             return;
         }
     }
-	
+
     // Update is called once per frame
     void Update()
     {
         Tile t = mouseController.GetMouseOverTile();
 
-        string s = "NULL";
+        if (t != null)
+        {
+            string s = "NULL";
+
+            if (t.furniture != null)
+            {
+                s = t.furniture.Name;
+            }
 
-        if (t.furniture != null)
+            myText.text = "Furniture: " + s;
+        }
+        else
         {
-            s = t.furniture.Name;
+            myText.text = "No Furniture";
```

Comment:
The function should return furniture type in text form. In that case i think "No Furniture" is more suitable. Its not that important anyway, becouse situation, where you hold mouse over no Tile rarely happens.